client/banner.js

```diff
@@ -12,78 +9,8 @@ import jQuery from 'jquery';
 import '../assets/stylesheets/banner.scss';
 
 jQuery( document ).ready( ( $ ) => {
-	$( '.woocommerce-services__connect-jetpack' ).one( 'click', function( event ) {
-		event.preventDefault();
+	$( '.woocommerce-services__connect-jetpack' ).one( 'click', function() {
 		const btn = $( this );
 		btn.addClass( 'disabled' );
-
-		installStep()
-			.then( activateStep )
-			.then( connectStep )
```

Comment:
We don't need any of these steps on the banner anymore?

Reply:
Nope! In the past the flow was AJAX-driven and led through the installation, activation, and connection via Jetpack. Now that we no longer depend on Jetpack, all we do is a single redirect.